var/spack/repos/builtin/packages/py-bluepyopt/package.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack.package import *
+
+
+class PyBluepyopt(PythonPackage):
+    """Bluebrain Python Optimisation Library"""
+
+    homepage = "https://github.com/BlueBrain/BluePyOpt"
+    pypi = "bluepyopt/bluepyopt-1.9.27.tar.gz"
+
+    # NOTE : while adding new release check pmi_rank.patch compatibility
+    version("1.14.4", sha256="7567fd736053250ca06030f67ad93c607b100c2b98df8dc588c26b64cb3e171c")
+
+    # patch required to avoid hpe-mpi linked mechanism library
+    patch("pmi_rank.patch")
+
+    variant("neuron", default=True, description="Use BluePyOpt together with NEURON")
```

Comment:
```suggestion
    variant("neuronml", default=True, description="Use BluePyOpt together with NEURON")
```
To match extra name

Reply:
not sure if I am missing something from this comment but `neuron` and `neuroml` are different packages. And hence variant needs to be remain same.

Reply:
After internal discussion we'll just remove the variant - neuron is an actual dependency (that's being corrected in https://github.com/BlueBrain/BluePyOpt/pull/477 )
`neuroml` doesn't appear to be a spack package yet, so for now it might be better to avoid this variant completely.